image/image.go

```diff
@@ -36,23 +36,31 @@ func IDFromDigest(digest digest.Digest) ID {
 type V1Image struct {
 	// ID is a unique 64 character identifier of the image
 	ID string `json:"id,omitempty"`
-	// Parent is the ID of the parent image
+	// Parent is the ID of the parent image.
+	//
+	// Depending on how the image was created, this field may be empty.
 	Parent string `json:"parent,omitempty"`
-	// Comment is the commit message that was set when committing the image
+	// Comment is an optional message that can be set when committing or
+	// importing the image.
 	Comment string `json:"comment,omitempty"`
 	// Created is the timestamp at which the image was created
 	Created time.Time `json:"created"`
-	// Container is the id of the container used to commit
+	// Container is the id the container that was used to create the image.
+	//
+	// Depending on how the image was created, this field may be empty.
 	Container string `json:"container,omitempty"`
 	// ContainerConfig is the configuration of the container that is committed into the image
 	ContainerConfig container.Config `json:"container_config,omitempty"`
-	// DockerVersion specifies the version of Docker that was used to build the image
+	// DockerVersion is the version of Docker that was used to build the image.
+	//
+	// Depending on how the image was created, this field may be empty.
 	DockerVersion string `json:"docker_version,omitempty"`
-	// Author is the name of the author that was specified when committing the image
+	// Author is the name of the author that was specified when committing the
+	// image, or as specified through MAINTAINER (deprecated) in the Dockerfile.
 	Author string `json:"author,omitempty"`
 	// Config is the configuration of the container received from the client
 	Config *container.Config `json:"config,omitempty"`
-	// Architecture is the hardware that the image is built and runs on
+	// Architecture is the hardware CPU architecture that the image is built and runs on.
```

Comment:
Doh, now I see -- I guess this is where that "built and runs on" verbiage came from? :grimacing: :see_no_evil: :innocent: